pkg/tsdb/cloudmonitoring/time_series_query.go

```diff
@@ -20,6 +20,19 @@ import (
 	"github.com/grafana/grafana/pkg/tsdb/intervalv2"
 )
 
+func (timeSeriesQuery cloudMonitoringTimeSeriesQuery) appendGraphPeriod(req *backend.QueryDataRequest) string {
+	if timeSeriesQuery.GraphPeriod != "disabled" {
```

Comment:
i think a note here explaining that graphPeriod: disabled and graphPeriod: "" are intentionally different could be useful for future versions of ourselves reading this code :D